modules/assisted-installer-assisted-installer-prerequisites.adoc

```diff
@@ -0,0 +1,60 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='assisted-installer-prerequisites_{context}']
+= Assisted Installer prerequisites
+
+Successful installation with the {ai-full} requires ensuring the following prerequisites. The installer will validate these prerequisites before installation.
+
+== Hardware
+
+For control plane nodes or the {sno} node, nodes must have at least:
+
+* 8 CPU cores
+* 16.00 GiB RAM
+* 100 GB storage
+* 10ms write speed or less for etcd `wal_fsync_duration_seconds`
+
+For worker nodes, each node must have at least:
+
+* 4 CPU cores
+* 16.00 GiB RAM
+* 100 GB storage
+
+== Networking
+
+The network must have:
+
+* A DHCP server unless using static IP addressing.
+* A base domain name. The base domain must have:
+  - You must ensure that there is no wildcard, such as `*.<cluster_name>.<base_domain>`, or the installation will not proceed.
+  - A DNS A/AAAA record for `api.<cluster_name>.<base_domain>`.
+  - A DNS A/AAAA record with a wildcard for `*.apps.<cluster_name>.<base_domain>`.
+* Port `6443` open for the API URL if you intend to allow users outside the firewall to access the cluster via the `oc` CLI tool.
+* Port `443` open for the console if you intend to allow users outside the firewall to access the console.
+
+[IMPORTANT]
+====
+DNS A/AAAA record settings at top-level domain registrars can take significant time to update. Ensure the A/AAAA record DNS settings are working before installation to prevent installation delays.
+====
+
+The {product-title} cluster's network must also have:
+
+* Connectivity between all cluster nodes
+* Connectivity for each node to the internet
+* Access to an NTP server for time synchronization between the cluster nodes
+
+== Pre-flight validations
+
+Before installing software on the nodes, the {ai-full} will:
+
+* Ensure network connectivity
+* Ensure sufficient network bandwidth
+* Ensure connectivity to the registry
+* Ensure time synchronization between cluster nodes
+* Verify that the cluster nodes meet the minimum hardware requirements
+* Validate the installation configuration parameters
+
+If the {ai-full} does not successfully validate the foregoing requirements, installation will not proceed. Ensuring the cluster meets the prerequisites before installation is beneficial, because it eliminates complex post-installation troubleshooting, thereby saving significant amounts of time and effort.
```

Comment:
I think this last sentence here would make a good introduction to this last list. Not to suggest something too marketing-speak, but maybe if you lead with the preflight validation as a feature to help you first (and that if it does not validate the installation doesn't proceed).